models/src/main/java/pm/gnosis/models/Wei.kt

```diff
@@ -4,11 +4,13 @@ import java.math.BigDecimal
 import java.math.BigInteger
 
 data class Wei(val value: BigInteger) {
-    fun toEther(scale: Int = 18): BigDecimal = BigDecimal(value).setScale(scale).div(BigDecimal(10).pow(18))
+    fun toEther(scale: Int = 18): BigDecimal = BigDecimal(value).setScale(scale).div(WEI_TO_ETHER_MULTIPLIER)
 
     fun toLong() = value.toLong()
 
     companion object {
+        private val WEI_TO_ETHER_MULTIPLIER = BigDecimal(10).pow(18)
         val ZERO = Wei(BigInteger.ZERO)
+        fun ether(value: Double) = Wei((BigDecimal.valueOf(value) * WEI_TO_ETHER_MULTIPLIER).toBigInteger())
```

Comment:
Not so sure about having `Double` here (ie.: in order to avoid loss of precision). Maybe receiving a `String` which represents the value or the `BigDecimal` itself 😉